Installers/Linux/RUN/postinstall.sh

```diff
@@ -88,6 +84,18 @@ case "$choice2" in
 	sudo -H -u $SUDO_USER bash -c 'mdtool setup install $DIR/Main/MonoDevelop.MonoGame.mpack'
 esac
 
+#MonoGame.Content.Builder.targets
+
+if [ ! -d /usr/lib/mono/xbuild/MonoGame ]; then
+	mkdir /usr/lib/mono/xbuild/MonoGame
+fi
+
+if [ ! -d /usr/lib/mono/xbuild/MonoGame/v3.0 ]; then
+	mkdir /usr/lib/mono/xbuild/MonoGame/v3.0
+fi
+
+sudo cp $DIR/Main/MonoGame.Content.Builder.targets tmp/usr/lib/mono/xbuild/MonoGame/v3.0/
```

Comment:
> Should this be tmp/usr/lib/..... ?

good cache

> Also with the tooling the XS/VS Templates and targets will expect the MGCB.exe (and its dependencies) to be in
> 
> /usr/lib/mono/xbuild/MonoGame/v3.0/Tools/

Will do that right away

> and the Assemblies to be in
> 
> /usr/lib/mono/xbuild/MonoGame/v3.0/Assemblies/<Platform>

What assemblies do you want me to include? Only Linux or something more? Also where do you think would be a good position to install them?